scripts/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  #  Create a service named db.
+  db:
+    image: "postgres:12-alpine"
+    container_name: "pipeline_db"
+    #   Setup the username, password, and database name
+    environment:
+      - POSTGRES_USER=pipeline
+      - POSTGRES_PASSWORD=pipeline
+      - POSTGRES_DB=pipeline
+    #   Map port 6432 (localhost) to port 5432 on the container
+    ports:
+      - "6432:5432"
```

Comment:
There is no need to expose the port for the `db` service to the host machine since it is not supposed to be reachable from the outer world. Only the application service should have a connection to it over the common [`network`](https://docs.docker.com/compose/compose-file/#networks).

Reply:
I need to know more about the planned infrastructure setup, is there any doc I could refer to? How will we debug (for example, db migrations) without connecting remotely to DB? I know there are ways to connect to container from the host it's running on, but it's more complicated than just creating a tunnel to the exposed port.

Reply:
Besides, for now there will be only db in that network :)